ivy/functional/frontends/torch/miscellaneous_ops.py

```diff
@@ -35,6 +35,11 @@ def roll(input, shifts, dims=None):
     return ivy.roll(input, shifts, axis=dims)
 
 
+@to_ivy_arrays_and_back
+def meshgrid(*tensors, indexing=None):
+    return ivy.meshgrid(*tensors, indexing=indexing)

```

Comment:
torch's meshgrid returns a tuple so cast the output to a tuple instead of a list, (this also for some reason fixes the type issues)